lib/person.rb

```diff
@@ -1 +1,80 @@
-# your code goes here
+class Person
+  attr_reader :name
+  attr_accessor :bank_account, :happiness, :hygiene
+
+  def initialize(
+    name,
+    bank_account = 25,
+    happiness = 8,
+    hygiene = 8
+    )
+    @name = name
+    @bank_account = bank_account
+    @happiness = happiness
+    @hygiene = hygiene
+  end
+
+  def happiness=(new_happiness)
+    if new_happiness > 10
+      @happiness = 10
+    elsif new_happiness < 0
+      @happiness = 0
+    else
+      @happiness = new_happiness
+    end
+  end
```

Comment:
Since you have `attr_accessor`s:
```suggestion
  def happiness=(new_happiness)
    if new_happiness > 10
      happiness = 10
    elsif new_happiness < 0
      happiness = 0
    else
      happiness = new_happiness
    end
  end
```

Reply:
And same throughout.